README.md

```diff
@@ -1,13 +1,47 @@
 # falcor-postman
 
-falcor-postman
+A tool that enables you to exercise your Express.js app Falcor's endpoint and therefore helps you to validate your queries.
```

Comment:
What about something shorter, i really like the graphiQL elevator pitch kind of description, we could borrow from it as we are doing the same thing only for falcor: 
"An graphical interactive in-browser Falcor IDE"